### PR TITLE
Added checkPassword functionality

### PR DESCRIPTION
**Pull request by Radio58**
https://trello.com/c/mTNHc2Wc/157-password-verification-checks

### CODE REVIEW

File: src/pages/register/index.js [hunk 1]
```diff
@@ -42,9 +42,10 @@ const Register = () => {
             type={"password"}
             />
           </form>
+          <p>{checkPassword(formData.password) ? 'Password OK!' : 'Password must contain at least one capital letter, one number and one special character'}</p>
```
Comment (Auenc) [inline src/pages/register/index.js:45]: This needs to use the translation system

**Review by Auenc — CHANGES_REQUESTED** [review]
Got some linting issues

**Review by Auenc — CHANGES_REQUESTED** [review]

File: src/pages/register/index.js (hunk 1 above)
Comment (Radio58) [reply src/pages/register/index.js]: This has been fixed

File: src/locales/en/translation.json [hunk 2]
```diff
@@ -84,5 +84,7 @@
   "welcome": "Welcome to Cohort Manager",
   "whatsOnYourMind": "What's on your mind?",
   "profileImageUrl": "Profile image URL",
-  "enterProfileImageUrl": "Enter your profile image URL"
+  "enterProfileImageUrl": "Enter your profile image URL",
+  "passwordOk": "Password is ok",
+  "invalidPassword": "Password must contain at least one capital letter, one number and one special character"
```
[the remark this answers is not in the document]
Comment (Radio58) [reply src/locales/en/translation.json:89]: Done

File: src/context/auth.js [hunk 3]
```diff
@@ -78,11 +78,24 @@ const AuthProvider = ({ children }) => {
     setLogout(true)
   }
 
-  const handleRegister = async (email, password) => {
-    const res = await register(email, password)
-    setToken(res.data.token)
+  const checkPassword = (password) => {
+    const hasUppercase = /[A-Z]/.test(password);
+    const hasNumber = /\d/.test(password);
+    const hasSpecialChar = /[ `!@#$%^&*()_+\-=\[\]{};':"\\|,.<>\/?~]/.test(password);
```
[the remark this answers is not in the document]
Comment (Radio58) [reply src/context/auth.js]: Fixed

**Review by Auenc — APPROVED** [review]

**Pull request merged**